iset.mm

```diff
@@ -61210,6 +61249,14 @@ would need to be strengthened (for example, to imply included middle) to
     ( wrel ccnv wfun cfn wcel w3a crn cdm df-rn relcnvfi 3adant2 simp2 syl5eqel
     fundmfi syl2anc ) ABZACZDZAEFZGZAHRIZEAJUAREFZSUBEFQTUCSAKLQSTMROPN $.
 
+  ${
+    $( If a 1-1 and onto function has a finite domain, its range is finite.
```

Comment:
"a bijection" ?

Reply:
We generally speaking use the "1-1" and "onto" terminology (with, I suppose, http://us.metamath.org/mpeuni/df-f1o.html as being the defining place) but I guess there's no reason to try to completely avoid "surjection" and "injection" and especially "bijection" :shrug: .